etc/spack/defaults/packages.yaml

```diff
@@ -46,7 +46,7 @@ packages:
       szip: [libszip, libaec]
       tbb: [intel-tbb]
       unwind: [libunwind]
-      sycl: [hipsycl]
```

Comment:
FYI: This was a duplicate from L45

Reply:
L45?

Reply:
It looks from a `git blame` like this was added unintentionally by https://github.com/spack/spack/pull/14682 and not caught in review. Grepping through the spack packages doesn't show any `depends_on('sycl')` so we can let this change through and see if anything bad happens?

Reply:
line 45. just up the page - it was a duplicated definition.

Reply:
https://github.com/spack/spack/pull/19087/files/d5510ed8c31d2590ac17560fc94199205cc12b9d#diff-50db37ac9e31f0e8d94666e865343cb0R45